### PR TITLE
Blog - An Ultimate Guide to Kubernetes Role-Based Access Control

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
@jalletto can you help with this error?

**Comment by jalletto** [comment]
@msali123 This error was caused by me. It should be fixed now. Make sure you pull the latest main branch and then push again and it should work. Let me know if you have any issues.

File: blog/_posts/2022-11-14-guide-rolebased-ctrl.md [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+
+Next, you have to create a ClusterBinding or a RoleBinding which binds a Kubernetes cluster user with a role. ClusterBinding is applied at a cluster level, while RoleBinding is applied at a namespace level.
+
+In this tutorial, you will learn how to implement RBAC at the cluster level and namespace level. In addition, you will learn how to use the declarative and imperative approach when implementing RBAC.
+
+## Prerequisites
+
+You will need a running cluster and Kubectl.
+ Basic knowledge of how service accounts work.
+
```
Comment (jalletto) [inline blog/_posts/2022-11-14-guide-rolebased-ctrl.md:28]:
```suggestion

* You will need a running cluster and Kubectl.
* Basic knowledge of how service accounts work.

```

File: blog/_posts/2022-11-14-guide-rolebased-ctrl.md [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+* `list`
+* `watch`
+* `create`
+* `update`
+* `patch`
+* `delete`
+
+The resources map states which resources and apiGroups the Role is being applied to:
+
+~~~{.yaml caption=">_"}
```
Comment (jalletto) [inline blog/_posts/2022-11-14-guide-rolebased-ctrl.md:45]:
```suggestion
~~~{.yaml caption=""}
```

**Review by jalletto — APPROVED** [review]

**Pull request merged**